nix/modules/scripts.nix

```
@@ -52,7 +52,7 @@
         # Want to update the root flake.lock if we're updating the version that is currently the default.
         if grep -qE "versions\.url = \".+\?dir=''${VERSIONS_DIR}\"" flake.nix; then
           # TODO, once the Nix version on CI supports it -> nix flake update versions
-          nix flake lock --tarball-ttl 0 --update-input versions --override-input versions "path:$VERSIONS_DIR"
+          nix flake update --tarball-ttl 0 versions
```

`--tarball-ttl 0` is (now?) a restricted option on my nix 2.22.1 multi-user install. 

this still works as a regular user:

```suggestion
          nix flake update --refresh versions
```

there's an unrelated occurrence further up that i cannot suggest changes for. i suggest replacing that too.

---

Done, and above